rust/tools/authorization-logic/src/main.rs

```diff
@@ -31,7 +31,7 @@ fn main() {
     let filename = &args[1];
     let in_dir = &args[2];
     let out_dir = &args[3];
-    let decl_skip = if(args.len() > 3) { &args[4] } else { "" };
+    let decl_skip = if(args.len() > 4) { &args[4] } else { "" };
```

Comment:
should this be 5 (assuming index of  `args` starts at `0`)?

Reply:
It seems it should be 4 not 5 because it's `>` rather than `>=`. I also just checked and it works at 4, but not at 5.

Reply:
Never mind. I thought this was an assert.